draft-ietf-scitt-architecture.md

```diff
@@ -614,103 +624,103 @@ The retrieved Receipt may be embedded in the corresponding COSE_Sign1 document i
 
 Due to the operational challenge of maintaining a globally consistent append-only Registry,
-some TS may provide limited support for historical queries on the Claims they have registered,
+some Transparency Services may provide limited support for historical queries on the Transparent Statements they have registered,
 and accept the risk of being blamed for inconsistent Registration or Issuer equivocation.
 
 Verifier and Auditors may also witness (1,4) but may not be able to collect verifiable evidence for it.
 
-**Availability of Transparent Signed Claims**
+#### Availability of Transparent Signed Statement
 
 Networking and Storage are trusted only for availability.
 
-Auditing may involve access to data beyond what is persisted in the TS log. For example, the registered TS may include only the hash of a detailed SBOM, which may limit the scope of auditing.
+Auditing may involve access to data beyond what is persisted in the Transparency Services. For example, the registered Transparency Service may include only the hash of a detailed SBOM, which may limit the scope of auditing.
 
 Resistance to denial-of-service is implementation specific.
 
-Actors should independently keep their own record of the Claims they issue, endorse, verify, or audit.
+Actors should independently keep their own record of the Signed Statements they issue, endorse, verify, or audit.
 
 ### Confidentiality and privacy.
 
-The network is untrusted. All contents exchanged between actors is protected using secure authenticated channels (TLS) but, as usual, this may not exclude network traffic analysis.
+According to Zero Trust Principles any location in a network is never trusted. All contents exchanged between actors is protected using secure authenticated channels (e.g., TLS) but, as usual, this may not exclude network traffic analysis.
 
-**Claims and their registration**
+#### Signed Statements and Their Tegistration
 
-The TS is trusted with the confidentiality of the claims presented for registration. Some TS may publish every claim in their logs, to facilitate their dissemination and auditing. Others may just return receipts to the client that present claims for registration, and disclose the ledger only to auditors trusted with the confidentiality of its contents.
+The Transparency Service is trusted with the confidentiality of the Signed Statements presented for registration. Some Transparency Services may publish every Transparent Statement in their logs, to facilitate their dissemination and auditing. Others may just return Receipts to clients that present Singed Statements for registration, and disclose the ledger only to auditors trusted with the confidentiality of its contents.
 
-A collection of transparent Claims leaks no information about the contents of other Claims registered at the TS.
+A collection of Transparent Statements leaks no information about the contents of other Transparent Statements registered at the Transparency Service.
```

Comment:
Is this describing a requirement on TSs, Issuers (users), or Verifiers? It's confusing.

Reply:
@nedmsmith, please see #51 to see if this clarifies things. If not, please open a new issue.